### PR TITLE
[2.0] Backports and release

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -93,9 +92,10 @@ jobs:
       test_extras: 'dev'
       test_command: 'pytest -p no:warnings --doctest-rst -m "not figure" --pyargs sunpy'
       submodules: false
+
       targets:
-        - wheels_linux
-        - wheels_macos
+        - wheels_cp3[78]-manylinux*x86_64
+        - wheels_cp3[78]-macosx*
```
Comment (nabobalis) [inline azure-pipelines.yml:98]: For the backport:

```suggestion
        - wheels_cp3[678]-manylinux*x86_64
        - wheels_cp3[678]-macosx*
```

**Comment by nabobalis** [comment]
Now to wait for the CI to pass.

**Comment by nabobalis** [comment]
Dev deps broken with MPL master.

File: setup.cfg [hunk 2]
```diff
@@ -95,6 +95,41 @@ remote_data_strict = True
...
+    ignore:direct construction of AsdfSchemaItem has been deprecated
+    ignore:direct construction of AsdfSchemaExampleItem has been deprecated
+    # See https://github.com/scikit-image/scikit-image/issues/4848
+    ignore:Converting `np.inexact` or `np.floating` to a dtype is deprecated
+    # See https://github.com/astropy/astropy/pull/10570
+    ignore:leap-second auto-update failed
+    # See https://github.com/astropy/extension-helpers/issues/23
+    ignore:Distutils was imported before Setuptools
+    # toolz internal deprecation warning https://github.com/pytoolz/toolz/issues/500
+    ignore:The toolz.compatibility module is no longer needed:DeprecationWarning
```
Comment (Cadair) [inline setup.cfg:131]: this got fixed upstream

**Review by Cadair — COMMENTED** [review]
probably should remove those pngs tho

**Comment by Cadair** [comment]
i feel like it would be useful to have the wheels build on prs to backport branches?

File: setup.cfg (hunk 2 above)
Comment (nabobalis) [reply setup.cfg]: removed it all

**Comment by nabobalis** [comment]
> i feel like it would be useful to have the wheels build on prs to backport branches?

Sure

**Comment by nabobalis** [comment]
> i feel like it would be useful to have the wheels build on prs to backport branches?

If you can work out the incantation for it.

**Comment by nabobalis** [comment]
> probably should remove those pngs tho

Removed.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,3 +1,76 @@
+Sunpy 2.0.2 (2020-09-26)
+========================
+
+Deprecations and Removals
+-------------------------
+
+- Removed deprecated functions:
```
Comment (Cadair) [inline CHANGELOG.rst:7]: why are we removing deprecated functions in a bugfix?

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,3 +1,76 @@
...
+  new JSON files. The old text files which the data used to come in no longer
+  exist. The new JSON files for `~sunpy.net.dataretriever.NOAAIndicesClient`
+  now do not have the following columns:
+  - Geomagnetic Observed and Smoothed
+  - Sunspot Numbers Ratio (RI/SW)
+
+  Both `sunpy.timseries.sources.NOAAIndicesTimeSeries` and
+  `sunpy.timseries.sources.NOAAPredictIndicesTimeSeries` have been updated to
+  support the new JSON files. Loading the old text files is still supported,
+  but support for this will be removed in a future version of sunpy. (`#4340 <https://github.com/sunpy/sunpy/pull/4340>`__)
```
Comment (Cadair) [inline CHANGELOG.rst:32]: I know this isn' t really relevant here but if we have deprecated this then it should be in the correct section. Also actually relevant here is we shouldn't backport deprecation warnings.

**Review by Cadair — CHANGES_REQUESTED** [review]
There are a lot more things in the changelog than there are in the diff.

**Comment by Cadair** [comment]
https://github.com/sunpy/sunpy/compare/v2.0.1...2.0#diff-6433478f47a9619c556daa50fe6af6afR129 this should not be in the release, we should never introduce new deprecation warnings in bugfixes.

This changelog entry https://github.com/sunpy/sunpy/compare/v2.0.1...2.0#diff-8275bb637288e8b6fa8d522da48ee460 dosen't belong and the diff dosen't seem to be in the release.

**Pull request merged**